test/e2e/storage/testsuites/podlogs.go

```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package podlogs enables live capturing of all events and log
+// Package testsuites enables live capturing of all events and log
 // messages for some or all pods in a namespace as they get generated.
 // This helps debugging both a running test (what is currently going
 // on?) and the output of a CI run (events appear in chronological
 // order and output that normally isn't available like the command
 // stdout messages are available).
-package podlogs
+package testsuites
 
 import (
```

Comment:
The functions in `podlogs.go` have nothing to do with the functions in the `testsuites` package. If the code really isn't welcome in `test/e2e/framework` anymore, then at least keep in a separate package. I suggest `test/e2e/storage/podlogs`.

Reply:
@pohly Thanks for your suggestion.
OK, I will make it as `test/e2e/storage/podlogs` today.

/hold

Reply:
done

/hold cancel